BitBar/version.12h.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+#
+# <bitbar.title>BitBar Version</bitbar.title>
+# <bitbar.version>v0.1.0</bitbar.version>
+# <bitbar.author>Olivier Tille</bitbar.author>
+# <bitbar.author.github>oliviernt</bitbar.author.github>
+# <bitbar.image>http://i.imgur.com/9BrFhSJ.png</bitbar.image>
+# <bitbar.desc>Checks the current BitBar version against the latest from GitHub</bitbar.desc>
+# <bitbar.dependencies>Ruby</bitbar.dependencies>
+#
+# BitBar version plugin
+# by Olivier Tille (@oliviernt)
+#
+# Checks the current BitBar version against the latest from GitHub
+
+require 'net/http'
+require 'json'
+require 'nokogiri'
+
+# create a new application at https://github.com/settings/developers and add client_id and client_secret here:
```

Comment:
You're absolutely right. I had that in there for development as I would hit the max request limit but that shouldn't be a problem when simply using the plugin.